src/util/graph.h

```diff
@@ -252,6 +252,16 @@ class grapht
   std::vector<node_indext>
   get_reachable(const std::vector<node_indext> &src, bool forwards) const;
 
+  std::vector<node_indext> depth_limited_search(
+    const node_indext &src,
+    std::size_t &limit,
```

Comment:
I believe @smowton has commented on this: not clear why this is passed by reference.